specs/IsPointInDraggableRegion.md

```diff
@@ -0,0 +1,303 @@
+    be duplicated here.
+
+    Examples:
+    * New set of classes and APIs (Custom Downloads):
+      https://github.com/MicrosoftEdge/WebView2Feedback/blob/master/specs/CustomDownload.md
+    * New member on an existing class (BackgroundColor):
+      https://github.com/MicrosoftEdge/WebView2Feedback/blob/master/specs/BackgroundColor.md
+
+    Style guide:
+    * Use second person; speak to the developer who will be learning/using this API.
+    (For example "you use this to..." rather than "the developer uses this to...")
+    * Use hard returns to keep the page width within ~100 columns.
+    (Otherwise it's more difficult to leave comments in a GitHub PR.)
+    * Talk about an API's behavior, not its implementation.
+    (Speak to the developer using this API, not to the team implementing it.)
+    * A picture is worth a thousand words.
+    * An example is worth a million words.
+    * Keep examples realistic but simple; don't add unrelated complications.
+    (An example that passes a stream needn't show the process of launching the File-Open dialog.)
+    * Use GitHub flavored Markdown: https://guides.github.com/features/mastering-markdown/
+
+-->
+
+Draggable Regions For Visual Hosted Apps
+===
+
+# Background
+<!-- TEMPLATE
+    Use this section to provide background context for the new API(s)
+    in this spec. Try to briefly provide enough information to be able to read
+    the rest of the document.
+
+    This section and the appendix are the only sections that likely
+    do not get copied into any official documentation, they're just an aid
+    to reading this spec. If you find useful information in the background
+    or appendix consider moving it to documentation.
+    
+    If you're modifying an existing API, included a link here to the
+    existing page(s) or spec documentation.
+
+    For example, this section is a place to explain why you're adding this
+    API rather than modifying an existing API.
+
+    For example, this is a place to provide a brief explanation of some dependent
+    area, just explanation enough to understand this new API, rather than telling
+    the reader "go read 100 pages of background information posted at ...". 
+-->
+Draggable regions are marked regions on a webpage that will move the app window when the user clicks and drags on them. It also opens up the system menu if right-clicked. This API is designed to provide draggable regions support for Visual Hosting.
+
+
+# Examples
+## Enable Dragging (L button)
+### Handle WM_NCHITTEST
+The objective is to tell Windows that the current mouse coordinate corresponds to a caption location if it is within a draggable region.
+#### Win32
+```cpp
+       case WM_NCHITTEST:
+            POINT pt {GET_X_LPARAM(lParam), GET_Y_LPARAM(lParam)};
+            BOOL isInRegion = FALSE;
+            webview2->IsPointInDraggableRegion(&pt, &isInRegion);
+
+            if(isInRegion)
+            return HTCAPTION;
```

Comment:
Microsoft sample code style guidelines say to use indenting and {} for single lines.
```suggestion
                return HTCAPTION;
            }
```